cpp/include/cudf/io/parquet.hpp

```diff
@@ -296,6 +317,19 @@ class parquet_reader_options_builder {
     return *this;
   }
 
+  /**
+   * @brief Sets enable/disable conversion of binary to strings per column.
+   *
+   * @param val Vector of boolean values to enable/disable conversion of binary to string columns.
+   * Note default is to convert to string columns.
+   * @return this for chaining
+   */
+  parquet_reader_options_builder& convert_binary_to_strings(std::vector<bool> val)
+  {
+    options._convert_binary_to_strings = val;
```

Comment:
Likewise here:
```suggestion
    options._convert_binary_to_strings = std::move(val);
```
`std::optional` does have a move constructor, but it'll need to be exercised with `std::move()`.